### PR TITLE
Format document using shfmt

### PR DESCRIPTION
**Pull request by MatheusRV**

### CODE REVIEW

File: tfb [hunk 1]
```diff
@@ -98,9 +98,9 @@ SCRIPT_PATH="$(realpath "$0")"
 SCRIPT_ROOT="$(dirname "$SCRIPT_PATH")"
 
 if ! docker network inspect tfb >/dev/null 2>&1; then
-  docker network create tfb >/dev/null
+    docker network create tfb >/dev/null
 fi
 
 test -t 1 && USE_TTY="-t"
-docker build -t techempower/tfb - < ${SCRIPT_ROOT}/Dockerfile
+docker build -t techempower/tfb - <${SCRIPT_ROOT}/Dockerfile
```
Comment (nbrady-techempower) [inline tfb]: This will break the docker build?

Comment (MatheusRV) [reply tfb]: No, this isnt a broken PR. Works perfectly.

Comment (nbrady-techempower) [reply tfb]: Sorry to be a nit but I prefer leaving the space between `<` and the `Dockerfile`. It's both easier to read and matches the docker documentation.

https://docs.docker.com/engine/reference/commandline/build/

image

**Comment by nbrady-techempower** [comment]
I'm going to close this for now. I'm ok with the double quoting but I'd prefer a space before and after redirection operators, like ` > somefile.txt`

**Pull request closed without merging**